### PR TITLE
ENH: Add step to rolling

### PR DESCRIPTION
**Pull request by bashtage**
- [ ] xref #227 
- [ ] Tests added: Please use `assert_type()` to assert the type of any return value

### CODE REVIEW

File: pandas-stubs/core/frame.pyi [hunk 1]
```diff
@@ -1763,26 +1764,30 @@ class DataFrame(NDFrame, OpsMixin):
     @overload
     def rolling(
         self,
-        window,
+        window: int,
```
Comment (Dr-Irv) [inline pandas-stubs/core/frame.pyi:1767]: docs say "int, offset, or BaseIndexer subclass"

**Review by Dr-Irv — CHANGES_REQUESTED** [review]
Need to clean up the type of `window`

File: pandas-stubs/core/frame.pyi (hunk 1 above)
Comment (bashtage) [reply pandas-stubs/core/frame.pyi]: Fixed.

**Review by Dr-Irv — APPROVED** [review]
thanks @bashtage

**Pull request merged**